lib/theme_check/checks/asset_size_css.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+module ThemeCheck
+  # Reports errors when trying to use too much JavaScript on page load
```

Comment:
Oops! 
```suggestion
  # Reports errors when trying to use too much CSS on page load
```

Maybe the comment is not needed now that we got some awesome doc?

Reply:
True that :p

Reply:
Copy paste forgot to change the comment. :sweat_smile: